jobs/kpi-forecasting/.gitignore

```diff
@@ -1,6 +1,12 @@
+# directories
 .cache
 .idea
+.ipython
+.ipynb_checkpoints
 .local
 .python-version
 .python_history
 .vscode
+
+# filetypes
+*.ipynb
```

Comment:
Could you also add `*.pyc` to the ignore list?

Reply:
done.